### DISCUSSION TITLE
Add GTA/Mafia Connected

### DISCUSSION THREAD

**Discussion by aWhhdGVteXNlbGY** [tltneon/lgsl#167] [Ideas]
https://gtaconnected.com/
https://mafiaconnected.com/

https://wiki.gtaconnected.com/ServerQuery
https://www.game-state.com/forum/viewtopic.php?f=9&t=5122

**Comment by tltneon** [comment]
Hello @ThatRoboticFish 
I added that protocol to https://github.com/tltneon/lgsl/tree/lgsl7
But for now it queries only basic server data, no players info

**Comment by aWhhdGVteXNlbGY** [comment]
Thanks for that.
BTW, GTAC also uses gtac://connect/[IP:PORT]/gta:[iii/vc/sa/iv/iv_eflc] for URLs and mafiac://connect/[IP:PORT]/mafia:[i] for MafiaC.
